modules/rgbd/samples/kinfu_demo.py

```diff
@@ -0,0 +1,60 @@
+import cv2
+import numpy as np
+
+from argparse import ArgumentParser
+
+def get_depth_list(folder):
+    if sys.platform.find('linux')+1:
+        f = open(folder + '/depth.txt', 'r')
+        rgb = [folder + '/' + s.replace('/', '/') for s in f.read().split() if '.png' in s]
+    else:
+        f = open(folder + '\\depth.txt', 'r')
+        rgb = [folder + '\\' + s.replace('/', '\\') for s in f.read().split() if '.png' in s]
+    return rgb
```

Comment:
Do we really need this branching?

Windows should work properly with forward slashes (/) too.

---

> `if '.png' in s`

`if s.endswith('.png')`

Reply:
done